Makefile

```diff
@@ -62,15 +62,17 @@ FULL_VERSION    := $(VERSION)
 endif
 
 
-DISTFILES	:= $(IN_FILES) $(CONF_FILES) Makefile
-
 INSTALL		:= install
 SED		:= sed
 SED_REPLACE	:= -e 's:@VERSION@:$(FULL_VERSION):g' \
 		-e 's:@sysconfdir@:$(sysconfdir):g' \
 		-e 's:@datadir@:$(datadir):g'
 
 
+ifeq ($(ARCH), s390x)
+CONF_FILES	+= features.d/qeth.modules features.d/dasd_mod.modules
+_dummy		:= $(shell ${SED} -i -e 's:features=":features="qeth dasd_mod :g' mkinitfs.conf)
+endif
```

Comment:
features.d/*.modules files could be added to CONF_FILES outside of this if-else block (the same place with others). But since we need to add to mkinitfs.conf anyway, I'd just add them all in this block.